internal/v1/api.yaml

```diff
@@ -2121,6 +2121,23 @@ components:
           type: integer
           description: Group id of the group to create (optional)
     User:
+      oneOf:
+        - $ref: '#/components/schemas/UserPassword'
+        - $ref: '#/components/schemas/UserKey'
+    UserPassword:
+      type: object
+      required:
+        - name
+        - password
+      properties:
+        name:
+          type: string
+          example: "user1"
+        password:
+          type: string
+          format: password
+          example: "my-secret-password"
```

Comment:
It does, but not on schema or object level on current openapi version, since 3.1 it is allowed on object level (now I would have to give examples array on top level like a request or response).

I could update the openapi to 3.1 but there is still open https://github.com/oapi-codegen/oapi-codegen/issues/373, not ideal since the generation doesn't support it.

So either I can:

- leave it as it is (example of plain password mentioning in description it won't be hashed again) - imho good enough for me until the openapi version allows it
- or create elaborate examples on level of every `requestBody` that delas with Customizations

Reply:
This is fine, it was just a brain dump. Let’s leave that idea.